documentation/src/docs/asciidoc/release-notes-5.0.0-M5.adoc

```diff
@@ -27,6 +27,8 @@ on GitHub.
 
 ===== New Features and Improvements
 
+* The JUnit Platform Gradle plugin task `junitPlatformTest` can be accessed
```

Comment:
Slight modification: "The JUnit Platform Gradle plugin task `junitPlatformTest` can now be accessed directly during the configuration phase of a build."